doc/quickstart/postgis_quickstart.rst

```diff
@@ -399,8 +401,8 @@ them blue for you - use the QGIS documentation to work out how to change this. Z
 a famous group of lakes in Canada.
 
 
-Creating A Spatial Table The Easy Way
-================================================================================
+Creating a spatial table the easy way
```

Comment:
please change the title: Importing spatial data into the database

Reply:
Done